crates/rpc/rpc-builder/src/lib.rs

```diff
@@ -1961,6 +2014,15 @@ struct WsHttpServer {
     jwt_secret: Option<JwtSecret>,
 }
 
+// Define the type alias with detailed type complexity
+type WsHttpServerKind = Server<
```

Comment:
`clippy` forced me to put this type in, 'cause it said creating `WsHttpServers`s with the entire server structure was illegal

Reply:
we want to get rid of these types entirely